src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,86 @@
+    axelard tx wasm instantiate 9 '{"service_registry_address":"axelar1hrpna9v7vs3stzyd4z3xf00676kf78zpe2u5ksvljswn2vnjp3ystlgl4x","service_name":"validators","source_gateway_address":"0xCa85f85C72df5f8428a440887CA7c449D94e0D0c","confirmation_height":1,"block_expiry":10,"source_chain":"test","rewards_address":"axelar1wkwy0xh89ksdgj9hr347dyd2dw7zesmtrue6kfzyml4vdtz6e5ws2pvc5e","voting_threshold":["9","10"]}' --from deployer --chain-id devnet-amplifier --gas auto --gas-adjustment 2 --gas-prices 0.007uamplifier --node http://devnet-amplifier.axelar.dev:26657 --label test-voting-verifier --no-admin
```

Comment:
This is a separate step to Instantiate the contract to make it more readable in the doc